### DISCUSSION TITLE
how to make a view file for homepage in an Theme module?

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#11661] [Q&A] [answered]
I made `Layout.cshtml` in   dir `Views` and it worked.

But `Index.cshtml` not work.

Which file shoul I make?

which controller is in charge of the view file?

**Comment by MikeAlhayek** [comment, marked as answer]
You’ll need to update the site setting by changing the HomeRoute. The HomeRoute defines which area,controller and action the user lands in first. So you can set it to where you want them to land which defines the view they’ll see

**Comment by ns8482e** [comment]
@scil you are using Razor Pages then place your file at `Pages\Index.cshtml`, set route to `@page "~/"`
If you are using MVC then create HomeController with `[Route('~/')]` and  place your file at ``Views\Home\Index.html`
